src/hooks/useUserAlerts.tsx

```diff
@@ -18,7 +18,7 @@ const createFrequentAlerts = (
   data: any
 ) => {
   return [
-    createOfferAlert(user, settings, data.user.offersReceived),
+    createOfferAlert(user, settings, data.user?.offersReceived || []),
```

Comment:
you can remove this [with my other pr](https://github.com/fxhash/fxhash-website/pull/656)